src/components/Navbar.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import { NavLink, Link } from "react-router-dom";
+import {UlStyled, LiStyled, InputTextStyled,ButtonTextStyled} from "./Navbar.style"
+
+class Navbar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Why are you using a state here if you don't update it

Reply:
I update it when I write in my input

Reply:
yes, you are right :)

Reply:
Thanks